src/openforms/tests/utils.py

```diff
@@ -55,14 +57,29 @@ def supress_output(stdchannel, dest_filename):
         dest_file.close()
 
 
+@contextlib.contextmanager
+def profile_outfile():
+    # On github CI, write to the step summary
+    github_summary = os.getenv("GITHUB_STEP_SUMMARY")
+    if not github_summary:
+        yield sys.stdout
+    else:
+        with Path(github_summary).open("a") as outfile:
+            outfile.write("```\n")
+            yield outfile
+            outfile.write("```")
+            outfile.write("\n\n")
```

Comment:
It is not too clear to me how we see the output data. If we set the `GITHUB_STEP_SUMMARY` var, it will be written to a file, otherwise it would endup in the standard output. But how do we see this file? I am probably missing something :smile:

Reply:
if you click on the "summary" in github actions in the top left, it's on the page!

![image](https://github.com/open-formulieren/open-forms/assets/5518550/11a123c7-d28c-4044-bcc8-71bb0d72f420)